tools_docker/README.md

```diff
@@ -57,24 +57,40 @@ have `agenda` app in `app.enabled` list, you MUST have `START_CALDAV=0`.
 If you want to use plugins and/or custom_toolbox you need to add files in `~/tracim/etc/plugins/` and `~/tracim/etc/custom_toolbox/` (default configuration). This two path are created when you start docker image for the first time.
 
 
+### Tracimcli inside docker
+
+For maintenance purpose you can use tracimcli command line in the docker this way:
+
+```bash
+docker exec -it -u www-data -w /etc/tracim {CONTAINER ID} tracimcli
```

Comment:
```suggestion
docker exec -i -u www-data -w /etc/tracim {CONTAINER ID} tracimcli
```

Reply:
t is not necessary there, but as this is interactive shell, it's a bit better with -t.
- prompt with -t : (tracimcli)
- prompt without -t is empty.

So i keep this one with -t.